Code/Tools/ProjectManager/Source/GemCatalog/GemInfo.h

```diff
@@ -75,8 +75,7 @@ namespace O3DE::ProjectManager
         QString m_version = "Unknown Version";
         QString m_lastUpdatedDate = "Unknown Date";
         int m_binarySizeInKB = 0;
-        QStringList m_dependingGemUuids;
-        QStringList m_conflictingGemUuids;
+        QStringList m_dependencies;
```

Comment:
These are gems names right? Are gem names unique? I don't think so but I do know that gem paths must be unique. This could potentially cause problems.

Reply:
Gem names are supposed to be unique - see enabled_gems.cmake